common/lib/capa/capa/util.py

```diff
@@ -12,6 +12,18 @@
 default_tolerance = '0.001%'
 
 
+def decimal_places(value):
+    """
+    count the "decimal_places" of "number". e.g, for
+    "number" with value "152.3667" the "decimal_places" will be
+    4 as there are 4 digits "3667" after decimal
+    """
+    decimal_places_count = -1
```

Comment:
why `-1`? Shouldn't we return `None` in this case? It will make for more readable code